src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/IS91DeterminationService.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonApiClient
+
+@Service
+class IS91DeterminationService(
+  private val prisonApiClient: PrisonApiClient
+) {
+
+  private object IS91Constants {
+    val resultCodes = setOf("5500", "4022", "3006", "5502")
+    const val offenceCode = "IA99000-001N"
+  }
+
+  fun getIS91AndExtraditionBookingIds(bookingIds: List<Long>): List<Long> {
```

Comment:
I considered making it `filter` but as it filters based off a call inside the method to the prison API, I thought leaving it as `get` may be better to signify that it's making an external API call.
I can change it to something `filter`y if you think it would be clearer though!

Reply:
Maybe it doing a request to an external api is an implementation detail. 
I'm not sure what a good `filter`y name would be so lets leave it!